feat: review Custom and Build workspace, move background Job to Build Workspace

FHenry opened the pull request:
Hope it will be more community accurate than  #17951 

Before 

![image](https://user-images.githubusercontent.com/1050053/187185190-56f948aa-49ea-4a33-9e35-e618823fa529.png)

![image](https://user-images.githubusercontent.com/1050053/187185231-5b911af4-1fa1-4da2-9b89-d33ebe502acf.png)

![image](https://user-images.githubusercontent.com/1050053/187185309-35295028-3747-497f-a6e3-50a9c877b9f1.png)


After

![image](https://user-images.githubusercontent.com/1050053/187187744-c25286a1-e717-4186-8567-df80d2df3c59.png)

![image](https://user-images.githubusercontent.com/1050053/187187787-83452849-f715-47a4-ab21-9ad7d0455ff7.png)




Note to reviewer :
Do not hesitate to tell me was can be acceptable in term of change. At first I just wish to add some Log into Core Workspace, but after digging I think it's can be rethink like that. 
But no problem if you think it's not a good things, it's a only a gentle proposal.

`no-docs`

FHenry committed:
feat: review link for Custom, build and NavBar

FHenry committed:
feat: review link for Custom, build and NavBar

FHenry committed:
feat: review link for Custom, build and NavBar

FHenry committed:
feat: review link for Custom, build and NavBar

FHenry committed:
feat: review link for Custom, build and NavBar

FHenry committed:
feat: review link for Custom, build and NavBar

FHenry committed:
Merge branch 'feat_review_default_admin_custom_link' of github.com:FHenry/frappe into feat_review_default_admin_custom_link

FHenry committed:
chore: linter json

FHenry committed:
fix: replace Block Module By Module Profile

FHenry committed:
chore: cypress test

ankush commented on package.json:
```diff
@@ -21,14 +21,20 @@
     "cookie": "^0.4.0",
     "cropperjs": "^1.5.12",
     "cssnano": "^5.0.0",
+    "cypress": "^6",
```
This doesn't seem intentional, can you undo these changes to pacakge.json?

FHenry replied:
Sorry, I was trying to run cypress test to understand the fail on cypress/integration/workspace_blocks.js, that's why there was this change, I reset to develop

FHenry committed:
chore: reset package.json to develop

FHenry committed:
Merge branch 'develop' into feat_review_default_admin_custom_link

FHenry committed:
Merge branch 'develop' into feat_review_default_admin_custom_link

ankush commented:
This looks much better / consistent... will do a few changes before merge 😬

ankush committed:
Merge branch 'develop' into feat_review_default_admin_custom_link

ankush commented:
Few changes:

- Workspace link is sorta redundant since they are created from workspace and visiting workspace list view isn't required. @shariquerik can confirm. 
- "System Logs" would be better than "Frappe Logs"

ankush committed:
refactor: workspace link, system logs section

- removed workspace listview link, visiting list view isn't required for
  practically all the tasks
- Frappe Logs -> System Logs

[skip ci]

shariquerik commented:
> * Workspace link is sorta redundant since they are created from workspace and visiting workspace list view isn't required. @shariquerik can confirm.

Not, required. It is only needed for custom apps to update the module field. for that also we have a Settings button for each workspace so no need for Workspace Link
